builtin/providers/aws/resource_aws_kms_key.go

```diff
@@ -170,6 +174,14 @@ func resourceAwsKmsKeyRead(d *schema.ResourceData, meta interface{}) error {
 	}
 	d.Set("enable_key_rotation", krs.KeyRotationEnabled)
 
+	tagList, err := conn.ListResourceTags(&kms.ListResourceTagsInput{
+		KeyId: metadata.KeyId,
+	})
+	if err != nil {
+		return nil
```

Comment:
As mentioned in https://github.com/hashicorp/terraform/issues/12137#issuecomment-287066725 we should not be ignoring the error here. Instead to avoid confusions you can do something similar to what I did in https://github.com/hashicorp/terraform/pull/12759 - i.e. make the error message descriptive enough.